.github/workflows/ci.yml

```diff
@@ -20,6 +20,17 @@ env:
   RUSTFLAGS: -Dwarnings
 
 jobs:
+  # Test and fmt the npm pkg
+  npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v4
+        with:
+            node-version: 20
+      - run: npm ci
+      - run: npm run fmt:check
+
```

Comment:
presumably this needs to cd into the relevant dir, but otherwise rad

Reply:
The subdir is `cargo-dist/templates/installer/npm` for reference